### PR TITLE
[tools/depends] bump python 3.9.10

### PR DESCRIPTION
**Pull request by fuzzard**
## Description
Bump python 3.9.10
Refresh patches as needed
use PYTHON3-VERSION file for version info

## Motivation and context
Stil in pre-alpha, may as well try and stay reasonably current

Have introduced the PYTHON3-VERSION as a minor. The intent for this is to introduce to all dependencies, and to look at introducing a full dependency build done via existing cmake system used for internal dependencies like ffmpeg. If anyone doesnt want this VERSION file change, just let me know, happy to pull out, but i dont think it really effects anyone to have it in.

## How has this been tested?
Runtime osx x86_64

## What is the effect on users?
N/A

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by fuzzard** [comment]
Jenkins build this please

File: tools/depends/target/python3/PYTHON3-VERSION [hunk 1]
```diff
@@ -0,0 +1,5 @@
+LIBNAME=Python
+VERSION=3.9.10
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.xz
+BASE_URL=https://www.python.org/ftp/python/$(VERSION)
```
Comment (wsnipex) [inline tools/depends/target/python3/PYTHON3-VERSION:5]: uploaded to mirrors

**Review by wsnipex — APPROVED** [review]
please don't forget to change the base url

**Comment by Maven85** [comment]
Hi @fuzzard,
since this PR Kodi master crashes on Android as soon as a program addon like "Kodi Android Installer" is opened.

Attached is a logcat of the crash.
[logcat_crash.txt](https://github.com/xbmc/xbmc/files/7969200/logcat_crash.txt)

**Comment by fuzzard** [comment]
Can you provide an addon that crashes on a non embedded platform? ie windows/macos
Is this android specific, arm v aarch64, this addon only etc?

**Comment by enen92** [comment]
I've been experiencing similar crashes on Kodi Android installer and HBO GO but only on Android, unfortunately not much more info available apart from identical crashlogs. Doesn't seem to happen on linux, but I'm not running 3.9.10 there (using 3.10.2 which has been solid for a few days)

**Comment by Maven85** [comment]
I don't have any problems on Windows either.

The problem seems to be "script.module.simplecache".
If I remove this from the Kodi Android Installer, Kodi no longer crashes when running this addon.

**Comment by CastagnaIT** [comment]
@fuzzard this python version, until the last 3.9.x available have a bug in sqlite
that cause a crash when you open two addons that make hard use kodi

Libreelec has already included a fix patch:
https://github.com/LibreELEC/LibreELEC.tv/pull/6345

Issues references:
https://github.com/LibreELEC/LibreELEC.tv/issues/6266
https://github.com/xbmc/xbmc/issues/21045

is it possible include this patch also in kodi build?

**Comment by fuzzard** [comment]
short answer is no. Reason is, 3.9.13 is due this weekend. Id rather, and indeed intend, to bump to that version which has the fix in it.

**Comment by CastagnaIT** [comment]
then even better thanks for the info

**Pull request merged**